pep_sphinx_extensions/pep_theme/templates/redirect.html

```diff
@@ -0,0 +1,16 @@
+{# Redirect template #}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="refresh" content="0; url={{ pathto(pep_file) }}">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PEP Redirector | peps.python.org</title>
+    <link rel="shortcut icon" href="{{ pathto('_static/py.png', resource=True) }}"/>
```

Comment:
```suggestion
```

As above. In particular, this element can trigger another HTTP request, which could add further delay to the redirect.